src/app/progress/SeasonalRank.tsx

```diff
@@ -43,12 +51,32 @@ export default function SeasonalRank({
 
   // Get seasonal character progressions
   const seasonProgress = characterProgressions.progressions[seasonPassProgressionHash];
-  const { level: seasonalRank, progressToNextLevel, nextLevelAt } = seasonProgress;
+  const prestigeProgress = characterProgressions.progressions[prestigeProgressionHash];
+
+  const prestigeMode = seasonProgress.level === 100;
```

Comment:
Does seaonProgress include a maximum level property?

Reply:
it does not